manage.py

```diff
@@ -46,10 +53,24 @@ def test(all=False):
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
 
+    q = Queue('default', connection=conn)
     while True:
         logging.info("Starting scrape...")
-        scrape_and_categorize_articles()
-        logging.info("Finished scrape. Let's run it back...")
+        job = q.enqueue(scrape_and_categorize_articles)
```

Comment:
I was thinking the one task we add ourselves to the queue is scrape articles, then as each article is pulled from the feed, we enqueue a categorization task for the one item.

Our old categorization relied on scraping and categorization happening in that order. If we enqueue after a scrape, then we don't need to worry about ```depends_on```.